contracts/ERC677BridgeTokenRewardable.sol

```diff
@@ -6,9 +6,9 @@ contract ERC677BridgeTokenRewardable is ERC677MultiBridgeToken {
     address public blockRewardContract;
     address public stakingContract;
 
-    constructor(string _name, string _symbol, uint8 _decimals)
+    constructor(string _name, string _symbol, uint8 _decimals, uint256 _chainId)
```

Comment:
In order to reduce the delta for introduced changes do you think it makes sense to have only `ERC677BridgeTokenRewardable` token contract to be permittable? So, the hierarchy of the token contracts could be:
-- ERC677BridgeToken
----PermittableToken
------ERC677MultiBridgeToken
--------ERC677BridgeTokenRewardable

Otherwise the changes you suggest will require the corresponding modifications in the tokenbridge monorepo, unit and e2e tests there assume that the deployment of the ERC677BridgeToken token does not require to specify the chain id.

Reply:
Sure, I will change the inheritance.

Reply:
This is done